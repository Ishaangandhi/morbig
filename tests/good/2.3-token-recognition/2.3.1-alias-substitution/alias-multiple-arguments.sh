alias show=echo list=ls
show hello
list
