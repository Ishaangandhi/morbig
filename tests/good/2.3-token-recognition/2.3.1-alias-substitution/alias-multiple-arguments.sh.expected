[
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "alias" ] ],
                [
                  "CmdSuffix_CmdSuffix_Word",
                  [
                    "CmdSuffix_CmdSuffix_Word",
                    [ "CmdSuffix_Word", [ "Word", "show=echo" ] ],
                    [ "Word", "list=ls" ]
                  ],
                  [ "Word", "coocoo=kiki" ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "echo" ] ],
                [ "CmdSuffix_Word", [ "Word", "hello" ] ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName",
                [ "CmdName_Word", [ "Word", "ls" ] ]
              ]
            ]
          ]
        ]
      ]
    ]
  ]
]